crates/virtio-bindings/src/lib.rs

```diff
@@ -1,24 +1,15 @@
 // Copyright 2019 Red Hat, Inc. All Rights Reserved.
 // SPDX-License-Identifier: (BSD-3-Clause OR Apache-2.0)
 
-#[cfg(all(feature = "virtio-v4_14_0", not(feature = "virtio-v5_0_0")))]
-mod bindings_v4_14_0;
-#[cfg(feature = "virtio-v5_0_0")]
-mod bindings_v5_0_0;
+#![allow(clippy::all)]
```

Comment:
I don't think this works as you intend it to work, which is also why the tests are failing. We want to allow clippy::all only in the bindings, now you are actually allowing it on the `virtio_blk` module which is the one that is immediately after the clippy macro.

Reply:
Ah, thank you!

Reply:
Actually I think this was correct.  My understanding is that `#!` applies to the whole of the current module, whereas `#` applies only to the next item.

The tests were failing with normal Rust warnings, not clippy ones, because I had accidentally dropped this line:

https://github.com/rust-vmm/vm-virtio/blob/c5895a61f6a4306d9167f33a7dba81ce29c7b001/crates/virtio-bindings/src/bindings_v5_0_0/mod.rs#L8-L9

Now that I've added it back as well, they're passing.